relay_handler.js

```diff
@@ -358,6 +376,8 @@ function handleFrameLazily(frame) {
     // - v2.Types.CallRequestCont
     var self = this;
 
+    var now = self.channel.timers.now();
```

Comment:
we don't have self.end on either of the lazy reqs; I'm already unhappy with how many fields they've gained.